src/main/java/com/jcabi/log/ColorfullyFormatted.java

```diff
@@ -63,8 +63,8 @@ public ColorfullyFormatted(final String bas, final String col) {
      */
     @Override
     public String format() {
-        return this.basic.replace(
-            new ControlSequenceIndicatorFormatted("%s?m").format(),
+        return this.basic.replaceAll(
+            new ControlSequenceIndicatorFormatted("%s\\?m").format(),
```

Comment:
Since "%s?m" is going to be treated as a regex, we have to escape the "?" character, which has, by default, special meaning.